scripts/check_doc_links.py

```diff
@@ -103,6 +103,7 @@ def main() -> None:  # pylint: disable=too-many-locals
         "https://github.com/valory-xyz/open-autonomy/trunk/infrastructure",
         "https://gateway.autonolas.tech/ipfs/`<hash>`,",
         "https://gateway.autonolas.tech/ipfs/Qmbh9SQLbNRawh9Km3PMEDSxo77k1wib8fYZUdZkhPBiev",
+        "https://encyclopedia.pub/entry/2959",
```

Comment:
Please revert, this link is not broken.

Reply:
If you open the link in browser or run the check locally it'll pass but it fails in the CI https://github.com/valory-xyz/open-autonomy/actions/runs/3296395745/jobs/5436004038 @dvilelaf tried running the CI multiple times but the CI was failing so we decided to skip it